CreditReport.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+class CreditReport {
```

Comment:
please rename the file using your initial, RA_CreditReport.java, and move these 2 homework files from HW1 to correct folder, otherwise it will create merge conflicts

Reply:
Understood, xièxiè